setup.py

```diff
@@ -16,15 +17,10 @@
 
     keywords='yelp',
 
-    packages=[
-        'yelp',
-        'yelp.obj'
-    ],
+    packages=find_packages(exclude=('tests*',)),
 
     install_requires=[
-        'httplib2==0.9.2',
-        'oauth2==1.9.0.post1'
+        'httplib2',
+        'oauth2',
```

Comment:
Yeah, we've got a pretty wide leeway with both these lib versions, so they shouldn't be pinned.